source/includes/legacy/legacy-command-exceptions.rst

```diff
@@ -0,0 +1,3 @@
+For commands whose output includes ``{ ok: 0 }``, ``mongosh`` returns a
+consistent exception and omits the server raw output. The legacy
+``mongo`` shell returned output that varied for each command.
```

Comment:
[optional suggestion]

Consider updating statements about legacy mongo shell behavior to the present tense.

Reply:
The original content used past tense for the legacy shell, but will go ahead and change for now.